src/day-1-toy/tuples.py

```diff
@@ -22,7 +22,9 @@ def dist(a, b):
 
 # Write a function that prints all the values in a tuple
 
-# def print_tuple(...
+def print_tuple(t):
+    for i in t:
+        print(i)
 
 t = (1, 2, 5, 7, 99)
 print_tuple(t)  # Prints 1 2 5 7 99, one per line
```

Comment:
For the very last problem here re: declaring a tuple of 1 element, compare:
``` u1 = (1) ``` and ``` u2 = (1,)```